package.json

```
@@ -5,17 +5,20 @@
   "main": "bin/www",
   "scripts": {
     "start": "node bin/www",
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "nyc mocha --timeout 5000 --report lcovonly --exit -R spec $(find ./tests -name \"*.spec.js\" -not -path \"./node_modules/*\")"
   },
   "author": "Daniele Manni",
   "license": "ISC",
   "dependencies": {
     "body-parser": "^1.19.0",
+    "chai": "^4.2.0",
```

```suggestion
    "chai": "^4.2.0",
    "chai-http": "^4.3.0",
```

Devono essere aggiunte come **dev dependencies**